numba/cpython/unsafe/tuple.py

```diff
@@ -36,6 +36,9 @@ def codegen(context, builder, signature, args):
 @intrinsic
 def build_full_slice_tuple(tyctx, sz):
     """Creates a sz-tuple of full slices."""
+    if not isinstance(sz, types.IntegerLiteral):
+        raise errors.RequireLiteralValue(sz)
```

Comment:
Self reminder: need to test this in CUDA

Reply:
`NUMBA_CAPTURED_ERRORS="new_style" python -m numba.runtests numba.cuda.tests` passes all tests for me locally.